### DISCUSSION TITLE
Automating workflow - what tasks to start with?

### DISCUSSION THREAD

**Discussion by pPris** [TEAMMATES/teammates#11263] [Q&A]
Hi, I'm interested in contributing to the [automations project](https://github.com/TEAMMATES/teammates/wiki/Automation-Hero), posting here since prof redirected me here. I'm wondering what tasks should we start automating first?

I've tried creating a sample workflow (clearing stale issues and pr's using github's pre-ready action/stale repo). The [workflow file can be found here](https://github.com/pPris/test-workflow-automations/blob/main/.github/workflows/stale.yml) and i used [this repo](https://github.com/pPris/test-workflow-automations/issues) to run the workflow file if you'd like to take a look

**Comment by damithc** [comment]
@ChooJeremy (or others) any pointers for @pPris ?

**Comment by damithc** [comment]
@pPris I sent you a slack message regarding this
